.github/workflows/format-code.yml

```diff
@@ -9,15 +9,10 @@ jobs:
     name: 'Format code'
     runs-on: ubuntu-latest
     if: github.event.issue.pull_request != '' && contains(github.event.comment.body, '/format')
-    steps:
-      - name: 'Check fork status'
-        id: fork_check
-        run: |
-          IS_FORK="$(jq '.pull_request.head.repo.fork' "$GITHUB_EVENT_PATH")"
-          echo "::set-output name=fork::$IS_FORK"
-          
+    steps:          
       - name: 'Setup SSH deploy key'
-        if: steps.fork_check.outputs.fork == "false"
+        # This will need to be adjusted when PRs are run against forks of the v3 repo
```

Comment:
In other words, if I want this to work on my own fork `SaschaMann/v3`, I'd have to change this to `SaschaMann`